### PR TITLE
API allows visitor can read a specific article 

### PR DESCRIPTION
**Pull request by emtalen**
# [PT Story](https://www.pivotaltracker.com/story/show/171976201)

## Changes proposed in this pull request:
* Creating show method for getting a single article in response
* Testing that functionality
* Adding Json Parse module
* Adding coverage to .gitignore

## What I have learned working on this feature:
* Repetition is the mother of all learning
* How to make a Json parse module

### CODE REVIEW

**Review by dernathul — APPROVED** [review]

File: app/controllers/api/v1/articles_controller.rb [hunk 1]
```diff
@@ -3,4 +3,10 @@ def index
     articles = Article.all
     render json: { articles: articles },status: 200
   end
+
+  def show
+    article = Article.find(params[:id])
+
```
Comment (pgaunitz) [inline app/controllers/api/v1/articles_controller.rb]: remove empty line

File: coverage/.last_run.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
   "result": {
```
Comment (pgaunitz) [inline coverage/.last_run.json]: this file should not be here

File: coverage/.resultset.json [hunk 3]
```diff
@@ -1,51 +1,57 @@
 {
```
Comment (pgaunitz) [inline coverage/.resultset.json]: this file should not be here

**Review by pgaunitz — CHANGES_REQUESTED** [review]

File: app/controllers/api/v1/articles_controller.rb (hunk 1 above)
Comment (emtalen) [reply app/controllers/api/v1/articles_controller.rb]: fixed this

File: coverage/.last_run.json (hunk 2 above)
Comment (emtalen) [reply coverage/.last_run.json]: I have added the coverage to the gitignore, which was not done before, I do not know how to remove these files now...

File: coverage/.resultset.json (hunk 3 above)
Comment (emtalen) [reply coverage/.resultset.json]: I have added the coverage to the gitignore, which was not done before, I do not know how to remove these files now...

**Review by pgaunitz — APPROVED** [review]

**Pull request merged**